docs/use_cases/recommender_systems.md

```diff
@@ -95,15 +94,17 @@ it increased recommendation accuracy by over 10%.
 
 #### Numerical data as side information
 
-A natural question: what to do in case the side information is numerical, not categorical? In this case we advise either of the following two:
+A natural question: what to do when the side information is numerical, not categorical? With numerical side information, we advise following one of the following two approaches::
 
-1. Inject a dummy user, with weights being scaled numerical values
+1. Inject a dummy user, with scaled numerical values for weights
 2. Analyze the distribution of the numerical data, and build categories based on value ranges
 
 
 ### Comparison with LightFM and Neural Collaborative Filtering
```

Comment:
previous comment - not yet addressed:

Comparison between what and LightFM and Neural Collaborative Filtering?
or do you mean "Comparison of LightFM and Neural Collaborative Filtering?"